servers/go-blocking/main.go

```diff
@@ -38,17 +46,29 @@ func main() {
 			log.Println("failed to accept connection:", err)
 			continue
 		}
+		nc := atomic.AddInt32(&numConns, 1)
+		if int(nc) >= numThreads {
+			numThreads *= 2
```

Comment:
GOMAXPROCS is not number of threads, it's just *maximum* number of threads to run goroutine.
(o/t There are another few threads).

So when number of client is 200, only 200 or less threads are running for the connection.